scripts/homestead.rb

```diff
@@ -265,8 +265,11 @@ def self.configure(config, settings)
               s.inline = 'ln -sf /opt/zray/zray.ini /etc/php/7.2/fpm/conf.d/zray.ini'
             end
           else
-            config.vm.provision 'shell' do |s|
+            # For proxy type sites, "to" is a port number
+            if site['to'].is_a? String
```

Comment:
Do nothing when the port is not a string is incorrect, it is going to silently fail and lead to unexpected behavior. What if instead you cast always to string?

I have tried this with `8080` and `"8080"` and work as expected:

```ruby
s.inline = 'rm -rf ' + site['to'].to_s + '/ZendServer'
```